app/controllers/admin_controller.rb

```diff
@@ -51,8 +51,10 @@ def projects
       c << ['LOWER(identifier) LIKE ? OR LOWER(name) LIKE ?', name, name]
     end
 
-    @projects = Project.find :all, order: 'lft',
-                                   conditions: c.conditions
+    @projects = Project.with_required_storage
+                .find :all,
+                      order: 'lft',
+                      conditions: c.conditions
```

Comment:
More like a general question:

This will most likely collide with the Rails 4 PR (since `find :all, ...` is long deprecated and those occurences were replaced for the Rails 4 migration):
Should we convert such statements to `where(c.conditions).order('lft')` on the spot, because we are touching them anyway? Or should we leave them as-is?